releases/reference-assemblies/README.md

```diff
@@ -12,6 +12,69 @@ The following scenarios and benefits are enabled for .NET Framework projects:
 
 Early preview.
 
+## Usage
+
+At the moment, packages are located on a non-default package feed. Until packages are moved to the default package feed, a change in your project's `NuGet.Config` file is needed. Add the following line to `<packageSources>` section:
+
+```xml
+<add key="roslyn-tools" value="https://dotnet.myget.org/f/roslyn-tools/api/v3/index.json" />
+```
+
+Sample `NuGet.Config` file:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+<packageSources>
+    <add key="roslyn-tools" value="https://dotnet.myget.org/f/roslyn-tools/api/v3/index.json" />
+</packageSources>
+</configuration>
+```
+
+Make the following changes based on the type of project you are using.
+
+### SDK-style project
+
+Add `TargetFramework` property and a `PackageReference` like in the following example `csproj` file:
```

Comment:
```suggestion
Add a `TargetFramework` property and a `PackageReference` like in the following example *csproj* file:
```

Reply:
There can only be one TargetFramework property but many PackageReference entries, so I think we should either keep it as is or use something like 'Add the TargetFramework...'. Do you have a current example of this property in docs?

Reply:
It's prefaced either with "a" or "the" in the doc set. In any case, there needs to be an article before TargetFramework. Either "the" or "a" is fine.

Reply:
I've updated the PR.